src/pages/index.js

```diff
@@ -128,4 +162,63 @@ addCardButton.addEventListener("click", () => {
 const userInfo = new UserInfo({
   userName: ".profile__title",
   userDescription: ".profile__description",
+  userPicture: ".profile__pic",
+});
+
+/* -------------------------------------------------------------------------- */
+/*                                     API                                    */
+/* -------------------------------------------------------------------------- */
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "63bd0a97-87e8-4761-93e7-bc9458ee6ee5",
+    "Content-Type": "application/json",
+  },
 });
+
+api.getUserInfo();
```

Comment:
Needs correction: don't forget to add `then` block where you will update the user info elements